tests/Game/Model/Node.cpp

```diff
@@ -0,0 +1,73 @@
+#include "PekingExpress/Game/Model/Connection.h"
+#include "PekingExpress/Game/Model/Node.h"
+#include <gtest/gtest.h>
+
+using namespace PekingExpress;
+
+class TestNode : public testing::Test
+{
+public:
+	TestNode() = default;
+	~TestNode() = default;
+protected:
+};
+
+TEST_F(TestNode, CreateEmptyNode_NodeCorrectlyCreated)
+{
+	const Node node = Node(0, {}, false);
+	const std::vector<Connection> connections = { };
+
+	EXPECT_EQ(0, node.GetId());
```

Comment:
`EXPECT_EQ` bij de andere `EXPECT_EQ` plaatsen (lijn 23). Dit is wat overzichtelijker en consistenter.